DESIGN.md

```diff
@@ -0,0 +1,71 @@
+# TLS Provider Design
+
+The TLS Provider offers a small surface area compared to other providers (like
+[AWS](https://registry.terraform.io/providers/hashicorp/aws/latest),
+[Google](https://registry.terraform.io/providers/hashicorp/google/latest),
+[Azure](https://registry.terraform.io/providers/hashicorp/azurerm/latest), ...),
+and focuses on covering the needs of working with entities like
+keys and certificates, that are part of
+[Transport Security Layer](https://en.wikipedia.org/wiki/Transport_Layer_Security).
```

Comment:
Transport Security Layer => Transport Layer Security

Reply:
🤦‍♀️ thank you